br/pkg/lightning/mydump/region.go

```diff
@@ -271,6 +272,10 @@ func makeSourceFileRegion(
 	if !isCsvFile {
 		divisor += 2
 	}
+	sizePerRow, err := SampleAndGetAvgRowSize(&fi, cfg, ioWorkers, store)
```

Comment:
`sizePerRow` is not a precise value. What if the number of rows exceeds the row id range?

Reply:
Still WIP, PTAL https://github.com/buchuitoudegou/tidb/commit/e5a9e5d6651bd41948e54a74eb873a5e6abd8b7f

Should I combine them into one or file another PR.

Reply:
Don't put it in another PR. Otherwise, this PR may introduce correctness issues.

Reply:
Will cherry-pick to this PR later.